### DISCUSSION TITLE
how to change termux-change-repo to readline mode

### DISCUSSION THREAD

**Discussion by a15489815-gmail** [termux/termux-app#3134] [General]
how to change termux-change-repo to readline mode ?

in debian it can achieve by debconf package . i can not install it in termux
![image](https://user-images.githubusercontent.com/120014792/206406902-781bb347-9a05-4bb4-80dc-c0492466421e.png)

i want to like this 

https://www.reddit.com/r/termux/comments/wftrea/changerepo_for_termux_readline_mode_working_only/

![image](https://user-images.githubusercontent.com/120014792/206407486-7898d223-8956-44ee-9024-8ebb5c745662.png)

**Comment by sylirre** [comment]
How to change it? Write own implementation.

Our variant is based on `dialog` utility and I don't see problems with using it in interactive sessions. If you need to replace mirror non-interactively, you will have to implement mirror changing on your own. This can be done by either echoing predefined mirror to a sources.list or picking URL from files in `$PREFIX/etc/termux/mirrors`.
